### PR TITLE
Feature/ui namespace wizard

### PR DESCRIPTION
**Pull request by PGuimarais**
- Implements 'Add Namespace' wizard
- Adds namespace wizard redux store

Bamboo Build:
http://builds.cask.co/browse/CDAP-DRC4527

### CODE REVIEW

File: cdap-ui/app/cdap/components/AdminConfigurePane/index.js [hunk 1]
```diff
@@ -14,20 +14,34 @@
...
  */
 
-import React from 'react';
+import React, {PropTypes} from 'react';
 require('./AdminConfigurePane.less');
 import ConfigureButton from '../ConfigureButton/index.js';
 var shortid = require('shortid');
 
-export default function AdminConfigurePane(){
+var configButtons = ['View Configurations', 'Add Namespace', 'Delete Namespace', 'Manage Namespaces', 'Instance Preference', 'Add Roles', 'Manage Roles', 'Reset Instance', 'Delete All Datasets', 'Reset Instance', 'Delete All Datasets', 'View Invalid Transactions', 'Tag Management'];
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/AdminConfigurePane/index.js]: This should be coming from `i18n` right?

File: cdap-ui/app/cdap/components/Wizard/WizardStepContent/WizardStepContent.less [hunk 2]
```diff
@@ -15,7 +15,9 @@
  */
 
 .wizard-step-content-container {
-  height: 100%;
+
+  height: 420px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Wizard/WizardStepContent/WizardStepContent.less]: This affects every single wizard. We should not do this.

File: cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  componentWillReceiveProps({isOpen}) {
+    this.setState({
+      showWizard: isOpen
+    });
+  }
+
+  createNamespace(){
+    let state = AddNamespaceStore.getState();
+
+    console.log('just created a namespace');
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js]: Remove?

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**